tests/e2e/testfiles/powerscale-templates/powerscale-auth-secret-template.yaml

```diff
@@ -1,9 +1,11 @@
 isilonClusters:
   - clusterName: "REPLACE_CLUSTERNAME"
-    username: "REPLACE_USER"
-    password: "REPLACE_PASS"
+    username: "-"
+    password: "-"
     isDefault: true
     endpoint: "REPLACE_AUTH_ENDPOINT"
-    endpointPort: "REPLACE_PORT"
+    endpointPort: REPLACE_AUTH_PORT
```

Comment:
Here we have it as REPLACE_AUTH_PORT with no quotation...

Reply:
Thought quotations was causing issues but it doesn't. With or without quotes it works but it was last verified without.